Game/src/Scenes/GameplayScene.cpp

```diff
@@ -220,37 +220,30 @@ void GameplayScene::Render(double delta)
     }
 
     if (m_drawCombo && std::get<7>(scores) > 0) {
-        m_amplitude = 30.0;
-        m_wiggleTime = 60 * m_comboTimer;
+        const double amplitudeStart = 30.0; // Constants
+        const double decrement = 1.0;
+        const double animationSpeed = 60.0;
 
-        double decrement = 6.0;                           // Calculate the decrement for each frame
-        double totalDecrement = decrement * m_wiggleTime; // Calculate the total decrement based on the current frame/time
-
-        double currentAmplitude = m_amplitude - totalDecrement; // Adjust the amplitude based on the total decrement
-
-        if (currentAmplitude < 0.0) {
-            currentAmplitude = 0.0; // Stopper like this due compiler issues
-        }
-
-        if (m_comboTimer > 1.0) {     // Check if the previous animation hasn't finished
-            m_comboTimer = 0.0;       // Reset the timer
-            m_amplitude += decrement; // Increase the starting amplitude for the next animation
+        if (m_comboTimer >= 1.0) { // Animation finished, reset parameters for the next animation
```

Comment:
move resetter to the top logic code so it will be correctly working as i intended